<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbgParamFittingWidget</class>
 <widget class="QWidget" name="DbgParamFittingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1551</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">DbgParamFittingWidget</string>
  </property>
  <widget class="QFrame" name="ParamFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1551</width>
     <height>831</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="QuantityCaliFrame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>1541</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="QuantityCaliRB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>801</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>定量校准</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="PtCntAndSegCntFrame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>1541</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="MajorCurvePtCntLab">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>主标曲点数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MajorCurvePtCntEdit">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="SegCntLab">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>分段段数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SegCntEdit">
     <property name="geometry">
      <rect>
       <x>1240</x>
       <y>20</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="ConcSgTbl">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>140</y>
      <width>768</width>
      <height>410</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>46</number>
    </attribute>
   </widget>
   <widget class="QTableView" name="StartEndTbl">
    <property name="geometry">
     <rect>
      <x>778</x>
      <y>140</y>
      <width>768</width>
      <height>410</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>46</number>
    </attribute>
   </widget>
   <widget class="QFrame" name="QualityCaliFrame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>560</y>
      <width>1541</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="QualityCaliRB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>761</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>定性校准</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="QualityFrame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>615</y>
      <width>1541</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="FormulaLab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>CutOff值计算公式</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FormulaVal1Edit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="FormulaVal2Edit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="FormulaVal3Edit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="FormulaVal4Edit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="FormulaVal5Edit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="MulSymbolLab">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>60</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AddSymbolLab">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>25</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="MulSymbol2Lab">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="MulSymbolLab_4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>) +</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="LotLab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>批号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LotEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>161</width>
       <height>40</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="YinYangJudgeLab">
     <property name="geometry">
      <rect>
       <x>778</x>
       <y>0</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>阴阳性判断方式</string>
     </property>
    </widget>
    <widget class="QLabel" name="Judge1StartTextLab">
     <property name="geometry">
      <rect>
       <x>778</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>当S/Co</string>
     </property>
    </widget>
    <widget class="QLabel" name="Judge2StartTextLab">
     <property name="geometry">
      <rect>
       <x>778</x>
       <y>110</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>当S/Co</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Judge1CndEdit">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>60</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Judge1EndTextLab">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>60</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>时，判断为阳性</string>
     </property>
    </widget>
    <widget class="QLabel" name="Judge2EndTextLab">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>110</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>时，判断为阴性</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Judge2CndEdit">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>110</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="BracketLab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1549</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1</width>
     <height>792</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>792</y>
     <width>1549</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1900</x>
     <y>0</y>
     <width>1</width>
     <height>812</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
